### DISCUSSION TITLE
QFieldCloud's ip addresses for PostGIS access

### DISCUSSION THREAD

**Discussion by nicdub** [opengisch/QField#3872] [Q&A]
Hello,

We have a QFieldCloud project which uses a PostGIS db. We had to open the db to the world as explained in the [doc](https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/), it's working fine. But I would prefer to be able to limit access to the db to QFieldCloud servers only. Do you have an ip list or range we could use?

Best,
Nicolas

**Comment by suricactus** [comment]
Hey, @nicdub . Please find the IP which QFieldCloud runs here: https://docs.qfield.org/reference/qfieldcloud/specs/ . Note this will most probably change in the future, but you can consider it stable for the next few months.

**Reply by nicdub** [reply]
Hey @suricactus, thank you for your prompt response!
